device_services/meta/parameters.md

```diff
@@ -174,9 +174,12 @@ Parameters service allows for advanced configuration of a device. For a list of
+| Parameter ID              | Name                    | Value Type | Requirements                              | Description                                                                                                                                                                                                                                      | 
+|---------------------------|-------------------------|------------|-------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `heat_control_mode`       | Heat control mode       | `string`   | `thermostat` or `power_regulator` service | Allows to switch heating control mode between thermostat and power <br/>regulator. Accepted values are `reg` for power regulator and `tht` for thermostat. Switch results in sending new inclusion report with requested device functionalities. |
+| `cable_always_locked`     | Cable always locked     | `bool`     | `chargepoint` service                     | Allows to enforce charging cable to be always locked on the EVSE side to prevent theft.                                                                                                                                                          |
+| `free_charging`           | Free charging           | `bool`     | `chargepoint` service                     | Determines whether chargepoint should automatically start charging an EV without an authorization or remote control trigger.                                                                                                                     |
+| `led_brightness`          | LED brightness          | `int`      | `chargepoint` service                     | Allows to set the brightness of the LED ring on the chargepoint. Accepted values are defined by `min` and `max` properties of the parameter definition.                                                                                          |
+| `device_lock`             | Device lock             | `int`      | `thermostat` or `power_regulator` service | Configure lock setting of the heating device.                                                                                                                                                                                                    |
+| `used_sensors`            | Used sensor             | `int`      | `thermostat` service                      | Set sensor type,which will be use to calculate heater setting                                                                                                                                                                                    |
```

Comment:
```suggestion
| `used_sensors`            | Used sensor             | `int`      | `thermostat` service                      | Select which sensor to use. This will be use to calculate heater setting.                                                                                                                                                                                    |
```